documentation/book/con-kafka-connect-configuration.adoc

```diff
@@ -0,0 +1,100 @@
+[id=kafka_connect_configuration_json_config-{context}]
+= Kafka Connect configuration
+
+The `spec.config` object of the `KafkaConnect` and `KafkaConnectS2I` resources allows detailed configuration of Apache Kafka Connect.
+This object should contain the Kafka Connect configuration options as keys. The values could be in one of the following JSON types:
+
+
+* String
+* Number
+* Boolean
+
+The `config` object supports all Kafka Connect configuration options with the exception of options related to:
+
+* Security (Encryption, Authentication and Authorization)
+* Listener or REST interface configuration
+* Plugin path configuration
+
+Specifically, all configuration options with keys starting with one of the following strings will be ignored:
+
+* `ssl.`
+* `sasl.`
+* `security.`
+* `listeners`
+* `plugin.path`
+* `rest.`
+
+All other options will be passed to Kafka Connect. A list of all the available options can be found on the
+http://kafka.apache.org/11/documentation.html#connectconfigs[Kafka website]. An example `config` field is provided
+below.
+
+.Example Kafka Connect configuration
+[source,json]
+----
+apiVersion: {KafkaConnectApiVersion}
+kind: KafkaConnect
+metadata:
+  name: my-connect-cluster
+spec:
```

Comment:
This example focuses only one one particular field (`KafkaConnect.spec.config`). But the user cannot take it and deploy it. Will it be understandable to the user? Is this how it is used in other Red Hat documents? When I do this I sometimes add `...` to make it clear that there will be more in the actual resource, such as:

```
spec:
  ...
  config:
    xxx:yyy
  ...
```

But I don't really know what is the best way.

Reply:
@scholzj can you add this to the agenda for our next sync up to talk about together?